integ/test-config.sh

```diff
@@ -5,7 +5,7 @@
 
 # Accept SSPL license to install MongoDB
 #   - If not set to `true` here, the Repository test component will fail immediately on execution
-export USER_ACCEPTS_SSPL_FOR_RFDK_TESTS=false
+export USER_ACCEPTS_SSPL_FOR_RFDK_TESTS=true
```

Comment:
Is this intentional? AFAIK, we leave the default value to false so that a user running it has to explicitly set the value to true.

Reply:
It was changed by mistake. Fixed.